Automattic/wp-calypso
# Activity Log: Add analytics to filter selection

Add tracks tracking to the filterbar everytime the user selects something. 

## to test
Set the `localStorage.setItem('debug', 'calypso:analytics:*');` so that you see the analtyitcs debug info.

Test out that date range selection filtering produces an event.
Test out that action type selection filtering produces an event.
Test out that resetting the filters produces an event.
